cloud: new generation of TiDB Cloud support services - Connected Care (#19958)
pingcap/docs: tidb-cloud/connected-care-detail.md

Reviewer: preview: https://pr.pingcap-docsite-preview.pages.dev/tidbcloud/connected-care-detail

---
title: Connected Care Details
summary: Introduces detailed information about the Connected Care support services for TiDB Cloud.
---

# Connected Care Details

This document provides detailed information about the [Connected Care](/tidb-cloud/connected-care-announcement.md) support services.

<table>
<thead>
  <tr>

>     <th></th>

Reviewer: ```suggestion
    <th>Support plan</th>
```

---
title: Connected Care Details
summary: Introduces detailed information about the Connected Care support services for TiDB Cloud.
---

> #Connected Care Details

Reviewer 2: ```suggestion
# Connected Care Details
```

    <td>Ticket-based technical support</td>
  </tr>
  <tr>
    <td>-</td>
    <td>End-to-end database guidance</td>
    <td>End-to-end database guidance</td>
    <td>End-to-end database guidance</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: AI chat in IM</td>
    <td>Connected: AI chat in IM</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: IM subscription for Cloud alerts</td>
    <td>Connected: IM subscription for Cloud alerts</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: IM subscription for support ticket updates</td>
    <td>Connected: IM subscription for support ticket updates</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: IM ticket creation</td>
    <td>Connected: IM ticket creation</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: Clinic service for advanced monitoring</td>
    <td>Connected: Clinic service for advanced monitoring</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>-</td>
    <td>Connected: IM interaction for support tickets</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>-</td>
    <td>Technical Account Management</td>
  </tr>
  <tr>

>     <td rowspan="5">Customer Resource</td>

Reviewer 2: ```suggestion
    <td rowspan="5">Customer Resources</td>
```

---
title: Connected Care Details
summary: Introduces detailed information about the Connected Care support services for TiDB Cloud.
---

# Connected Care Details

This document provides detailed information about the [Connected Care](/tidb-cloud/connected-care-announcement.md) support services.

<table>
<thead>
  <tr>
    <th></th>
    <th>Basic</th>
    <th>Developer</th>
    <th>Enterprise</th>
    <th>Premium</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>Features and Services</td>
    <td>Great for personal starter projects, connecting you with an active community and AI-powered assistance for essential support needs.</td>
    <td>Ideal for development and testing phases, offering direct connections and unlimited access to tech support, to kickstart your cloud journey.</td>
    <td>Designed for production workloads, connecting your operations with advanced monitoring, prompt alerts, and fast 7x24 response times for a seamless experience.</td>
    <td>Built for enterprises with mission-critical workloads, offering the fastest response times, proactive support, and dedicated connections through named Technical Account Managers.</td>
  </tr>
  <tr>

>     <td>Pricing</td>

Reviewer 3: Same - let's remove the pricing details.

    <td>Free</td>

>     <td>Monthly cost is based on the greater of $29 or 10% of total monthly charges.<br><br><strong>Sign up with a one-month free trial</strong></td>

Author: ```suggestion
    <td>Monthly cost is based on the greater of $29 or 10% of total monthly charges.</td>
```

>   <tr>
>     <td>Pricing</td>
>     <td>Free</td>
>     <td>Monthly cost is based on the greater of $29 or 10% of total monthly charges.</td>
>     <td>Monthly cost is based on the greater of $699 or 20% of total monthly charges.</td>
>     <td>Contact sales</td>
>   </tr>

Reviewer: ```suggestion
```

>     <td>Monthly cost is based on the greater of $699 or 20% of total monthly charges.<br><br><strong>Sign up, or contact sales</strong></td>

Author: ```suggestion
    <td>Monthly cost is based on the greater of $699 or 20% of total monthly charges.</td>
```

    <td>-</td>
    <td>S1: 4 business hours</td>
    <td>S1: 1 hour</td>
    <td>S1: 30 minutes</td>
  </tr>
  <tr>
    <td>-</td>
    <td>S2: 8 business hours</td>
    <td>S2: 2 hours</td>
    <td>S2: 1 hour</td>
  </tr>
  <tr>
    <td>-</td>
    <td>S3: 12 business hours</td>
    <td>S3: 8 business hours</td>
    <td>S3: 4 business hours</td>
  </tr>
  <tr>
    <td>-</td>
    <td>S4: 24 business hours</td>
    <td>S4: 24 business hours</td>
    <td>S4: 24 business hours</td>
  </tr>
  <tr>
    <td rowspan="10">Services</td>
    <td>Billing and account support</td>
    <td>Billing and account support</td>
    <td>Billing and account support</td>
    <td>Billing and account support</td>
  </tr>
  <tr>
    <td>-</td>
    <td>Ticket-based technical support</td>
    <td>Ticket-based technical support</td>
    <td>Ticket-based technical support</td>
  </tr>
  <tr>
    <td>-</td>
    <td>End-to-end database guidance</td>
    <td>End-to-end database guidance</td>
    <td>End-to-end database guidance</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: AI chat in IM</td>
    <td>Connected: AI chat in IM</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>

>     <td>Connected: IM subscription for Cloud alerts</td>
>     <td>Connected: IM subscription for Cloud alerts</td>

Reviewer: ```suggestion
    <td>Connected: IM subscription for TiDB Cloud alerts</td>
    <td>Connected: IM subscription for TiDB Cloud alerts</td>
```

  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: IM subscription for support ticket updates</td>
    <td>Connected: IM subscription for support ticket updates</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>

>     <td>Connected: IM ticket creation</td>

Reviewer 3: Please see if we are able to adjust the column length for better visualization. 
Currently, it looks like one takes two rows and the other takes only one row
<img width="483" alt="Screenshot 2025-01-14 at 2 48 52 PM" src="https://github.com/user-attachments/assets/ce161226-d83f-4146-8d79-b35f87669486" />

Reviewer: Let me create a preview of this PR and check it.

Reviewer: In the [preview](https://pr.pingcap-docsite-preview.pages.dev/tidbcloud/connected-care-detail), the column length looks the same.

![image](https://github.com/user-attachments/assets/6aeeb3bf-53a8-494d-aaa6-623e6bb02d24)

    <td>Connected: IM ticket creation</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>Connected: Clinic service for advanced monitoring</td>
    <td>Connected: Clinic service for advanced monitoring</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>-</td>
    <td>Connected: IM interaction for support tickets</td>
  </tr>
  <tr>
    <td>-</td>
    <td>-</td>
    <td>-</td>
    <td>Technical Account Management</td>
  </tr>
  <tr>
    <td rowspan="5">Customer Resources</td>
    <td>Documentations</td>
    <td>Documentations</td>
    <td>Documentations</td>
    <td>Documentations</td>
  </tr>
  <tr>
    <td>FAQ</td>
    <td>FAQ</td>
    <td>FAQ</td>
    <td>FAQ</td>
  </tr>
  <tr>
    <td>Community Workspace</td>
    <td>Community Workspace</td>
    <td>Community Workspace</td>
    <td>Community Workspace</td>
  </tr>
  <tr>
    <td>TiDB.AI</td>
    <td>TiDB.AI</td>
    <td>TiDB.AI</td>
    <td>TiDB.AI</td>
  </tr>
  <tr>
    <td>-</td>
    <td>Knowledge Base</td>
    <td>Knowledge Base</td>
    <td>Knowledge Base</td>
  </tr>
</tbody>
</table>

> - Business Hours: 9 AM - 6 PM local time, Monday to Friday

Reviewer 2: ```suggestion
- Business hours: 9 AM - 6 PM local time, Monday to Friday
```

> - IM: Instant Message. Currently, [Slack](https://slack.com/) and [Lark](https://www.larksuite.com/) are supported. In the future, Microsoft Teams is planned.

Reviewer: ```suggestion
- IM: Instant Message. Currently, [Slack](https://slack.com/) and [Lark](https://www.larksuite.com/) are supported. We are planning to support [Microsoft Teams](https://www.microsoft.com/en-us/microsoft-teams/group-chat-software) in the future.
```

> - Clinic: a modern monitoring service that provides advanced monitoring and diagnostic capabilities for your TiDB Cloud clusters.

Reviewer 2: Do we need to provide the Clinic link here https://docs.pingcap.com/tidb/stable/clinic-introduction#pingcap-clinic-overview?

Reviewer: It refers to Clinic on TiDB Cloud (https://github.com/pingcap/docs/pull/19895/files), which will be released along with the Connected Care service.

> - Clinic: a modern monitoring service that provides advanced monitoring and diagnostic capabilities for your TiDB Cloud clusters.

Reviewer 2: ```suggestion
- [Clinic](tidb-cloud/tidb-cloud-clinic.md): a modern monitoring service that provides advanced monitoring and diagnostic capabilities for your TiDB Cloud clusters.
```
